src/opensuse/15.2/helix/amd64/Dockerfile

```diff
@@ -28,12 +28,17 @@ RUN zypper ref && \
         python3-devel \
         python3-pip \
         sudo \
+        wget \
         unzip && \
         \
         zypper clean -a
 
 ENV LANG=en_US.utf8
 
+RUN wget https://github.com/microsoft/msquic/releases/download/v2.1.1/libmsquic-2.1.1-1.x86_64.rpm && \
+    rpm -i libmsquic-2.1.1-1.x86_64.rpm && \
```

Comment:
This seems to bypass signature checking.
```
rpm -i libmsquic-2.1.1-1.x86_64.rpm
warning: libmsquic-2.1.1-1.x86_64.rpm: Header V4 RSA/SHA256 Signature, key ID be1229cf: NOKEY
```
`zypper` can install packages directly via http without wget.
```
zypper install -y https://github.com/microsoft/msquic/releases/download/v2.1.1/libmsquic-2.1.1-1.x86_64.rpm
```

it fails on signature verification ... that is probably correct. My recommendation would be to add signing key

Reply:
I will probably need some hint about the signing key, I have never done that sort of thing before

Reply:
```
bde607ad2ca8:/ # rpm --import https://packages.microsoft.com/keys/microsoft.asc
bde607ad2ca8:/ # zypper install -y https://github.com/microsoft/msquic/releases/download/v2.1.1/libmsquic-2.1.1-1.x86_64.rpm
Loading repository data...
Warning: Repository 'Main Update Repository' appears to be outdated. Consider using a different mirror or server.
Warning: Repository 'Update Repository (Non-Oss)' appears to be outdated. Consider using a different mirror or server.
Reading installed packages...
Resolving package dependencies...

The following NEW package is going to be installed:
  libmsquic

1 new package to install.
Overall download size: 6.1 MiB. Already cached: 0 B. After the operation, additional 22.0 MiB will be used.
Continue? [y/n/v/...? shows all options] (y): y
Retrieving package libmsquic-2.1.1-1.x86_64                                                                                                                                                                      (1/1),   6.1 MiB ( 22.0 MiB unpacked)

Checking for file conflicts: ...................................................................................................................................................................................................................[done]
(1/1) Installing: libmsquic-2.1.1-1.x86_64 .....................................................................................................................................................................................................[done]
```

I would add same download + verify checksum + import logic we have elsewhere. 
This may be paranoid but prevents cases when somebody sneaks evil bits on the server.